srdb.class.php

```diff
@@ -953,6 +953,10 @@ public function replacer( $search = '', $replace = '', $tables = array( ) ) {
 
 						}
 
+						if ( empty( $update_sql ) ){
+							continue;
+						}
+
 						if ( $dry_run ) {
 							// nothing for this state
 						} elseif ( $update && ! empty( $where_sql ) ) {
```

Comment:
This $update check looks like it does the same thing as checking if $update_sql is empty.